.github/workflows/lint.yml

```
@@ -26,15 +26,16 @@ jobs:
+      - name: Install uv
         if: steps.changed-files.outputs.any_changed == 'true'
-      - uses: cachix/cachix-action@6a9a34cdd93d0ae4b4b59fd678660efb08109f2f # v12
+        run: curl -LsSf https://astral.sh/uv/install.sh | sh
+      - name: Add uv to PATH
         if: steps.changed-files.outputs.any_changed == 'true'
-        with:
-          name: mantrachain-e2e
-      - run: nix-shell -I nixpkgs=./nix -p test-env --run "make lint-py"
-        if: steps.changed-files.outputs.any_changed == 'true'
\ No newline at end of file
+        run: echo "$HOME/.cargo/bin" >> $GITHUB_PATH
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:1:28: Double quote to prevent globbing and word splitting

(shellcheck)